tiled/server/app.py

```diff
@@ -111,7 +115,7 @@ def build_app(
     deserialization_registry: Optional[SerializationRegistry] = None,
     compression_registry: Optional[CompressionRegistry] = None,
     validation_registry: Optional[ValidationRegistry] = None,
-    tasks=None,
+    tasks: Optional[dict[str, list[AppTask]]] = None,
```

Comment:
```suggestion
    tasks: Optional[TaskMap] = None,
```

How far away are we from being able to